packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -53,9 +53,9 @@ const ecsService = new ecs.Ec2Service(this, 'Service', {
 
 For a set of constructs defining common ECS architectural patterns, see the `@aws-cdk/aws-ecs-patterns` package.
 
-## AWS Fargate vs Amazon ECS
```

Comment:
Can we add ECS in here something like `## ECS running on AWS Fargate vs Amazon EC2`

Reply:
hi @pkandasamy91,  thanks for reply. 
I agree with your thought. it will be nice to mention about ECS.

But in table of contents, its has `Clusters`, `Task definitions`, `Service` ...  which are features of Amazon ECS. [docs](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/Welcome.html)

Therefore,  it could be a bit verbose to mention `ECS running on...` on strapline.

How about
`## Launch Types` or `## Launch Types: AWS Fargate vs Amazon EC2`
as they has own dedicated docs for it. [docs](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/launch_types.html)

Reply:
That sounds good to me!

Reply:
included `## Launch Types: AWS Fargate vs Amazon EC2` at new commit.

Reply:
Awesome! Thanks for updating this!